core/pubnub_grant_token_api.c

```diff
@@ -201,7 +201,7 @@ static CborError data_recursion(CborValue* it, int nestingLevel, char** json_res
                         encoded_sig.ptr[0] = '\0';
                     }
 
-                    char base64_str[encoded_sig.size + 2];
+                    char base64_str[67]; // HMAC+SHA256 max size + quotes + tailing null
```

Comment:
any reason for hardcoding here with 67?

Reply:
According to the [docs](https://www.pubnub.com/docs/sdks/rest-api#grant-access-v3) the `sig` is an HMAC+SHA256 value. Usually we got around 48 bytes but the maximum value of that key can be up to [64 bytes](https://crypto.stackexchange.com/questions/34864/key-size-for-hmac-sha256).